neqo-server/src/main.rs

```diff
@@ -466,6 +466,11 @@ impl ServersRunner {
                     max_table_size_decoder: args.max_table_size_decoder,
                     max_blocked_streams: args.max_blocked_streams,
                 },
+                Some(HttpZeroRttChecker::new(QpackSettings {
+                    max_table_size_encoder: args.max_table_size_encoder,
+                    max_table_size_decoder: args.max_table_size_decoder,
+                    max_blocked_streams: args.max_blocked_streams,
+                })),
```

Comment:
How is this different than `None`?

Reply:
It is not. I used that for testing and wanted to remove it, but forgot.